examples/chaiscript/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-ChaiScript)
+
+# DOCUMENTATION_START {
+hunter_add_package(chaiscript)
+find_package(chaiscript CONFIG REQUIRED)
+# DOCUMENTATION_END }
+
+if (CMAKE_HOST_UNIX) 
+    #set unix link libraries
+    list(APPEND LIBS "dl") 
+    list(APPEND LIBS "pthread")
+#    list(APPEND LIBS "readline")
+    add_definitions(-pthread)
+endif()
+if (CMAKE_HOST_WIN32) 
```

Comment:
Usually `CMAKE_HOST_*` is not a right variable to check, e.g. `CMAKE_HOST_UNIX` can be `TRUE` but `WIN32` can be `TRUE` too if toolchain is MinGW. Also I don't think we need this code at all since only Visual Studio toolchain is in CI configs.